app/assets/javascripts/application.js

```diff
@@ -32,9 +32,9 @@ $( document ).ready(function () {
   //   }
   // });
   
-  var places = $('.place');
+  var votesGraphs = $('.vote');
```

Comment:
You might consider naming variables that are jQuery located elements with a peso  ($).  e.g. `$votesGraphs`
This can serve as an indicator to future developers as to what this is.